Kernel/VM/MemoryManager.cpp

```diff
@@ -88,26 +91,133 @@ void MemoryManager::initialize_paging()
 
         if (mmap->type != MULTIBOOT_MEMORY_AVAILABLE)
             continue;
+
         // FIXME: Maybe make use of stuff below the 1MB mark?
         if (mmap->addr < (1 * MB))
             continue;
 
+#ifdef MM_DEBUG
+        kprintf("MM: considering memory at %p - %p\n",
+            (dword)mmap->addr, (dword)(mmap->addr + mmap->len));
+#endif
+
+        int in_super = 0, in_user = 0;
```

Comment:
This is C++ so let's use `bool` :)
Also one declaration per line please.

```suggestion
        bool in_super = false;
        bool in_user = false;
```